src/main/java/duke/Task.java

```diff
@@ -0,0 +1,45 @@
+package duke;
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
```

Comment:
I like how you named this boolean value to make it sounds like a boolean. 👍

Reply:
Agreed!